source/command_bus.coffee

```diff
@@ -7,16 +7,19 @@ class Space.messaging.CommandBus extends Space.Object
   }
 
   _handlers: null
+  _onSendHooks: null
 
   constructor: ->
     super
     @_handlers = {}
+    @_onSendHooks = []
 
   send: (command, callback) ->
     if @meteor.isServer
       # ON THE SERVER
       handler = @_handlers[command.typeName()]
-      if not handler?
+      hook(command) for hook in @_onSendHooks
```

Comment:
Now this is free from the testing concepts, I can see what we have :-)

The commandBus is providing the _onSend_ hook, which we register callbacks with. I'd name it like this:
`callback(command) for callback in @_onSendCallbacks`